compiler/noirc_evaluator/src/ssa/opt/basic_conditional.rs

```diff
@@ -0,0 +1,379 @@
+use std::collections::{HashSet, VecDeque};
+
+use acvm::AcirField;
+use fxhash::FxHashMap;
```

Comment:
```suggestion
use fxhash::FxHashMap as HashMap;
```
nit: we do this throughout the codebase, would be good to do here for consistency

Reply:
Done